NEWS.md

```diff
@@ -1,3 +1,34 @@
+### v11.3.0 (2023-10-23)
+
+#### Features
+
+* Updated agent initialization to allow running in worker threads when config.worker_threads.enabled is true ([#1817](https://github.com/newrelic/node-newrelic/pull/1817)) ([a39f0ef](https://github.com/newrelic/node-newrelic/commit/a39f0ef5ac670d03ab407b24e5aeccd8d5e8c680))
+
+#### Bug fixes
+
+* Updated Elasticsearch instrumentation to register on v7.13.0+ only ([#1816](https://github.com/newrelic/node-newrelic/pull/1816)) ([6437671](https://github.com/newrelic/node-newrelic/commit/6437671b921cd6bd73ed31180b0d62f62cc229a2))
+
+#### Security improvements

```

Comment:
This should be moved to chores as it's not a security fix for customers as it's a dev dep